src/wazuh_modules/vulnerability_scanner/doc/rtr.md

```diff
@@ -0,0 +1,194 @@
+# RTR
+This module, and its dependencies:
+- [router](../../../shared_modules/router/)
+- [indexer_connector](../../../shared_modules/indexer_connector/)
+- [content_manager](../../../shared_modules/content_manager/)
```

Comment:
don't we need to a reference to the `vulnerability_scanner` module? 🤔 
```suggestion
- [content_manager](../../../shared_modules/content_manager/)
- [vulnerability_scanner](../../../wazuh_modules/vulnerability_scanner/)
```

Reply:
Vulnerability scanner is the module itself